Dockerfile

```diff
@@ -40,7 +40,7 @@ ENV VPNSP=pia \
     PORT_FORWARDING=off \
     PORT_FORWARDING_STATUS_FILE="/forwarded_port" \
     # Mullvad only
-    COUNTRY=Sweden \
+    COUNTRY=sweden \

```

Comment:
It should be case insensitive right? Or did I forget to code that :eyes:

Reply:
I just pushed a small commit updating golibs so it treats all environment variables without case sensitivity (lowercase everything).